rma/security/ir.model.access.csv

```diff
@@ -8,3 +8,6 @@ access_rma_operation_user_own,rma.operation.user.own,model_rma_operation,rma_gro
 access_rma_operation_manager,rma.operation.manager,model_rma_operation,rma_group_manager,1,1,1,1
 access_rma_tag_user_own,rma.tag.user.own,model_rma_tag,rma_group_user_own,1,0,0,0
 access_rma_tag_manager,rma.tag.manager,model_rma_tag,rma_group_manager,1,1,1,1
+access_rma_finalization_portal,rma.finalization.portal,model_rma_finalization,base.group_portal,1,0,0,0
+access_rma_finalization_user_own,rma.finalization.user.own,model_rma_finalization,rma_group_user_own,1,1,1,0
```

Comment:
This should be okay for read. The reasons should be only created by managers.

Reply:
Done